gcc/rust/resolve/rust-ast-resolve-pattern.cc

```diff
@@ -38,49 +37,25 @@ PatternDeclaration::go (AST::Pattern *pattern, Rib::ItemType type,
 {
   PatternDeclaration resolver (bindings, type);
   pattern->accept_vis (resolver);
+
+  for (auto &[ident, info] : resolver.missing_bindings)
```

Comment:
this is C++17 style we only allow up to c++11 this is likely why the gcc4.8 build failed too